packages/mermaid/src/diagrams/class/parser/classDiagram.jison

```diff
@@ -127,6 +128,10 @@ line was introduced with 'click'.
 <*>\-                           return 'MINUS';
 <*>"."                          return 'DOT';
 <*>\+                           return 'PLUS';
+":"                             return 'COLON';
+","                             return 'COMMA';
+\#                              return 'BRKT';
+"#"                             return 'BRKT';
```

Comment:
It's potentially worth removing `#` from https://github.com/mermaid-js/mermaid/blob/b92b344844256287f038c7455210ca8546f9f308/packages/mermaid/src/diagrams/class/parser/classDiagram.jison#L141 too, but if it currently works, that's also okay.

Reply:
Leaving as is for now